spec/shared_behaviors/taggable.rb

```diff
@@ -22,6 +22,11 @@ def tag_link(s)
 
     it "supports non-ascii characters" do
       expect(@object.tags(true).map(&:name)).to include('vöglein')
+      expect(@object.tags(true).map(&:name)).to include('ինչո՞ւ')
+      expect(@object.tags(true).map(&:name)).to include('այո՜ո')
+      expect(@object.tags(true).map(&:name)).to include('սեւ֊սպիտակ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.